<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastrarCarro</class>
 <widget class="QDialog" name="CadastrarCarro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Carros</string>
  </property>
  <widget class="QLineEdit" name="txtPlaca">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>AAA-9999</string>
   </property>
   <property name="placeholderText">
    <string>Placa...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbCor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMarca">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbModelo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlaca">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>52</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Placa</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>52</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cor</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMarca">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>52</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblModelo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>52</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Modelo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbEstado">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbCidade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblUf">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>52</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UF</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCidade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>52</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cidade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCadastrar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/img/Save.png</normaloff>:/img/Save.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddModelo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Modelo</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/img/1_1-128.png</normaloff>:/img/1_1-128.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddCor">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Cor</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/img/1_1-128.png</normaloff>:/img/1_1-128.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddCidade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Cidade</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/img/1_1-128.png</normaloff>:/img/1_1-128.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCadastrar</sender>
   <signal>clicked()</signal>
   <receiver>CadastrarCarro</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
